javascript/javascript2/week1/hyf-bay/main.css

```
@@ -0,0 +1,38 @@
+body {
+    font-family: "Open Sans", sans-serif;
+    background-color: #F9FBFD;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body, h1, h2 {
+    margin: 0;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+.flex-row{
```

Try to be consistent in adding a space before curly brackets
```suggestion
.flex-row {
```